### ISSUE TITLE
Full-screen overlay showed on warning

### ISSUE THREAD

**Issue by Alex-Sokolov** [webpack/webpack-dev-server#861]
**Do you want to request a *feature* or report a *bug*?**
Bug

**What is the current behavior?**
[Documentation says](https://webpack.js.org/configuration/dev-server/#devserver-overlay)

> Shows a full-screen overlay in the browser when there are compiler errors or warnings. Disabled by default. If you want to show only compiler errors:
> overlay: true

Showed error overlay for warning
![default](https://cloud.githubusercontent.com/assets/4497128/24392557/17dac360-139d-11e7-9e3e-8328a4b68b97.PNG)

**If the current behavior is a bug, please provide the steps to reproduce.**
- set `overlay: true` in webpack-dev-server config section
- set warning for console calls in eslint for example
- use console in code
- run app

**What is the expected behavior?**
No overlay

**Workaround:**
Set `overlay` to
```
overlay: {
  warnings: false,
  errors: true
}
```

**Please mention your webpack and Operating System version.**
Windows 8.1
Webpack 2.3.2
Webpack-dev-server 2.4.2

**Comment by petermikitsh** [comment]
I'm still getting overlay warnings when using:

```
    devServer: {
      overlay: {
        warnings: false,
        errors: true
      }
    }
```

In-browser screenshot:

<img width="454" alt="screen shot 2017-04-10 at 2 06 52 pm" src="https://cloud.githubusercontent.com/assets/1571918/24882731/fd2da0be-1df6-11e7-980f-8b6b57746651.png">

Console output:

```
WARNING in duplicate-package-checker:
  <webpack>
    2.2.1 ./~/@fuego/forge/~/webpack
    2.3.3 ./~/webpack

webpack: Compiled with warnings.
```

The console says it's a warning, but it's still showing up in the browser overlay.

**Comment by petermikitsh** [comment]
Also, I'm on `2.4.2` of webpack-dev-server.

**Comment by gabrielstuff** [comment]
Using: 
```
overlay: {
  warnings: false,
  errors: true
}
```

Fixed it with `2.4.5` version
